trello/table_trello_card.go

```diff
@@ -0,0 +1,354 @@
+			{
+				Name:        "pos",
+				Description: "The position of the card.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "short_link",
+				Description: "The shortened link of the card.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "short_url",
+				Description: "The shortened URL of the card.",
+				Transform:   transform.FromField("ShortURL"),
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "start",
+				Description: "The start time of the card.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "subscribed",
+				Description: "Indicates whether the card has been subscribed.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "url",
+				Description: "The URL of the card.",
+				Transform:   transform.FromField("URL"),
+				Type:        proto.ColumnType_STRING,
+			},
+
+			// JSON fields
+			{
+				Name:        "attachments",
+				Description: "The attachments of the card.",
+				Hydrate:     getCardAttachments,
+				Transform:   transform.FromValue(),
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "badges",
+				Description: "The badges of the card.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "check_item_states",
+				Description: "The check item states of the card.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "checklists",
+				Description: "The checklists of the card.",
+				Hydrate:     getCardChecklists,
+				Transform:   transform.FromValue(),
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "custom_field_items",
+				Description: "The custom field items of the card.",
+				Hydrate:     getCardCustomFieldItems,
+				Transform:   transform.FromValue(),
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "custom_field_map",
+				Description: "The custom field map of the card.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "id_check_lists",
+				Description: "The ids of checklists attached to the card.",
+				Hydrate:     getCard,
+				Transform:   transform.FromField("IDCheckLists"),
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "id_labels",
+				Description: "The ids of labels attached to the card.",
+				Transform:   transform.FromField("IDLabels"),
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "labels",
+				Description: "The labels of the card.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "id_members",
+				Description: "The ids of members attached to the card.",
+				Transform:   transform.FromField("IDMembers"),
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "id_members_voted",
+				Description: "The ids of members who voted on the card.",
+				Transform:   transform.FromField("IDMembersVoted"),
+				Type:        proto.ColumnType_JSON,
+			},
```

Comment:
The missing columns though available in the SDK are not being returned from the [API](https://developer.atlassian.com/cloud/trello/rest/api-group-cards/#api-cards-id-get). So to avoid any confusion I haven't added them as columns.